### DISCUSSION TITLE
High vegetation for TEB (new version)

### DISCUSSION THREAD

**Discussion by MelPoupelin** [orbisgis/geoclimate#880] [Q&A]
Hi everyone, 

One of the latest version of TEB (5.5.0 used here in Dijon) includes an urban canyon fraction surfaces calculation system that is different from the previous one : 
If before: **Road + Building + Low-veg + High_veg = 1**
Now we have: **Road + Building + Low-veg = 1**
We now consider that the fraction of high vegetation is a "cube" located above the roadway and/or the low vegetation, i.e. : **High_veg <= 1 - Building** 
_(@wurtzj will surely confirm or specify)_ 

This means that some information must be given about what is ****under** the foliage of the trees** (road or low vegetation). 

So, even if we take the current results from Geoclimate and apply a coefficient in order that : (Road + Building + Low_veg = 1), we are going to overestimate the building fraction. One solution could be to "fill in" the equivalent of high vegetation only with road and low_veg, but maybe there is an other way for GeoClimate to identify the type of soil under tree foliage within databases ? 

For example, if I give to GeoClimate the BD TOPO + a fine scale low and high vegetation database ?

Regards,
Mélissa

**Comment by wurtzj** [comment]
Hello @MelPoupelin, yes in TEB from SURFEXv9 High vegetation is apart from the rest of vegetatation to take into account trees over roads.
But this seems indeed hard to know and it needs some specific hypothesis. You can find more informations on the current hypothesis in the thesis of Emilie Bernard (Appendix) https://www.theses.fr/2021TOU30250.

For sure combining different database would be the best. @j3r3m1 may tell you more about this.

Kind Regards,

**Reply by j3r3m1** [reply]
Yes a script with the possibility to use any input data should come soon.

**Reply by j3r3m1** [reply]
Yes a script with the possibility to use any input data should come soon.

**Comment by j3r3m1** [comment]
Right, there are the superimpositions between high vegetation and anything else at TSU scale but not at grid scale. I think it would be nice to include this option We just have to define a default value here.
https://github.com/orbisgis/geoclimate/blob/ffa10ae81e498a75f33789f20c14b64aa3dbf167/geoindicators/src/main/groovy/org/orbisgis/geoclimate/geoindicators/WorkflowGeoIndicators.groovy#L1780

@ebocher what do you think ?

**Comment by j3r3m1** [comment]
In the superimposition calculations at grid scale, should we replace "BUILDING_FRACTION", "ROAD_FRACTION", "WATER_FRACTION", etc. by solely "LAND_FRACTIONS" ?
And we use the default values of the surfaceFraction function for both "superpositions" and "priorities" ?

**Reply by j3r3m1** [reply]
@ebocher
